packages/atlas/src/components/FilterButton/FilterButton.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { Meta, StoryFn } from '@storybook/react'
+import { useState } from 'react'
+
+import { SvgActionShoppingCart } from '@/assets/icons'
+
+import { FilterButton, FilterButtonProps } from './FilterButton'
+
+export default {
+  title: 'inputs/FilterButton',
+  component: FilterButton,
+  args: {
+    options: [
+      {
+        label: 'Small',
+        id: 'small',
+      },
+      {
+        label: 'Medium',
+        id: 'medium',
+      },
+    ],
+    label: 'Label',
+    icon: <SvgActionShoppingCart />,
+  },
+} as Meta<FilterButtonProps>
+
+const Template: StoryFn<FilterButtonProps> = (args) => {
+  const [value, setValue] = useState<number[]>([])
+  return <FilterButton {...args} selected={value} onApply={setValue} />
```

Comment:
I don't see the counter when I'm selecting items. 
Not sure, but this could be caused by the storybook. Could you make this work?

Reply:
Filters get applied after `Apply` click

Reply:
Oh yeah, my bad 👍🏻  - didn't check that